sdk/core/azure-core/azure/core/pipeline/transport/_aiohttp.py

```diff
@@ -267,7 +267,7 @@ def __init__(self, request: HttpRequest, aiohttp_response: aiohttp.ClientRespons
         self.status_code = aiohttp_response.status
         self.headers = CIMultiDict(aiohttp_response.headers)
         self.reason = aiohttp_response.reason
-        self.content_type = aiohttp_response.headers.get('content-type')
+        self.content_type = self.headers.get('content-type')
```

Comment:
callout: I think this should be `self.headers` instead of `aiohttp_response.headers`, bc we made `self.headers` case insensitive

Reply:
Technically, from a functional perspective, this shouldn't matter since `aiohttp_response.headers` is also case-insensitive.

Reply:
yeah, @lmazuel brought that up too. tbh, i'll just go ahead and delete it, it doesn't need to be there